openslides/agenda/projector.py

```diff
@@ -136,3 +138,9 @@ def get_collection_elements_required_for_this(self, collection_element, config_e
                         output.extend(self.get_requirements_as_collection_elements(config_entry))
                         break
         return output
+
+
+def get_all_slides() -> Generator[ProjectorElement, None, None]:
+    yield ItemListSlide()
+    yield ListOfSpeakersSlide()
+    yield CurrentListOfSpeakersSlide()
```

Comment:
Alternatively this could return Generator[Type[ProjectorElement], None, None] (the class and not the instance). Then get_all_projector_elements could return "new" instances on each call. What do you think?

Reply:
I think we do not need new instances on each call.